src/desktop/apps/article/components/layouts/Article.tsx

```diff
@@ -105,11 +106,12 @@ export class ArticleLayout extends React.Component<
       showTooltips,
       showCollectionsRail,
       areHostedAdsEnabled,
-      shouldAdRender,
+      articleSerial,
       templates: { SuperArticleFooter, SuperArticleHeader } = {} as any,
     } = this.props
 
     const isStatic = isSuper || article.seriesArticle || customEditorial
+    const renderAd = shouldAdRender(null, null, null, article.layout)
```

Comment:
I don't quite understand why the first 3 parameters get passed in as null. It seems like the below would always evaluate to `false`?

`Math.abs(position) % frequency === 0`

Reply:
I'm trying to use the same method for all article types to determine if we should/not show ads on it.  So this helper func was updated to accept the needed params for News Articles and now Standard and Feature articles as well. I'm passing null for the first 3 params b/c they aren't needed on these types.